src/services/api/galleries.js

```diff
@@ -0,0 +1,21 @@
+import { apiBase } from 'services/config'
+import { GALLERIES, SITES } from 'common/constants'
+
+const galleries = {
+  get: async ({ sitename, id }) =>
+    apiBase().get(`${SITES}/${sitename}/${GALLERIES}/${id}`).json(),
+  getAll: async ({ sitename }) =>
+    apiBase().get(`${SITES}/${sitename}/${GALLERIES}/`).json(),
+  create: async ({ sitename, properties }) =>
+    apiBase()
+      .post(`${SITES}/${sitename}/${GALLERIES}/`, { json: properties })
+      .json(),
+  update: async ({ sitename, id, properties }) =>
```

Comment:
Is there any need for a partial update/patch API call with galleries? The back end doe have that functionality but it might not be used

Reply:
Not sure yet. I'll see when I implement the ui and add it if required. Cheers